06_30_2018/trailingzero.py

```diff
@@ -0,0 +1,25 @@
+#!usr/bin/python
+
+#counts the number of trailing zero for n factorial
+
+import math
+def count_trailing_zero_n_fact(n):
+    n_factorial = str(math.factorial(n))
+    trailing_zero_count = 0
+    if n_factorial.endswith('0'):
+        for i in reversed(n_factorial):
```

Comment:
Calling `reversed(n_factorial)` on every iteration of the for loop can be confusing. You could do something like this.

```
reversed_factorial = reversed(n_factorial)
for digit in reversed_factorial:
   # do more
```